scripts/export.py

```diff
@@ -116,6 +119,14 @@ def nemo_export(argv):
         sys.exit(1)
     typecheck.set_typecheck_enabled(enabled=False)
 
+    if args.cache_support:
+        export_cache_support_prev = model.encoder.export_cache_support
+        model.encoder.export_cache_support = True
+        logging.warning("Caching support is enabled.")
```

Comment:
Why is it a warning? Info should be fine.

Reply:
made it info.